index.js

```diff
@@ -288,14 +299,29 @@ exports.evm2wast = function (evmCode, opts = {
         i = findNextJumpDest(evmCode, i)
         break
       default:
-        if (callbackFuncs.has(op.name)) {
+        if (USE_ASYNC_API && callbackFuncs.has(op.name)) {
```

Comment:
Should make `USE_ASYNC_API` part of `opts.useAsyncAPI` so we can build both from the commandline.